doc/news/changes.h

```diff
@@ -159,6 +159,12 @@ inconvenience this causes.
 <a name="general"></a>
 <h3>General</h3>
 <ol>
+  <li> New: Triangulation::ghost_neighbors() returns a list of MPI ranks of owners
```

Comment:
Well, it isn't a list but a vector. But that made me think about the fact that what you probably *really* want is a `std::set`.